content/2018-09-09-newsletter-11.md

```diff
@@ -0,0 +1,86 @@
++++
+title = "The Embedded Working Group Newsletter - 11"
+date = 2018-09-09
+draft = false
+in_search_index = true
+template = "page.html"
++++
+
+This is the eleventh newsletter of the [Embedded WG] where we highlight new progress, celebrate cool projects, thank the community, and advertise projects that need help!
+
+[Embedded WG]: https://github.com/rust-embedded/wg
+
+Discuss on [internals.rust-lang.org], [on twitter], or [on reddit]!
+
+<!-- TODO, update links after post lands -->
+[internals.rust-lang.org]: #
+[on twitter]: #
+[on reddit]: #
+
+<!-- more -->
+
+If you want to mention something in [the next newsletter], make sure to leave a comment on the issue.
+
+[the next newsletter]: https://github.com/rust-embedded/blog/issues/12
+
+## Highlights
+
+* `rustc` now supports 4 **Cortex-R** targets thanks to the work of [paoloteti] in the Cortex-R space! You can build programs for these targets using nothing but the Rust toolchain
+* [James Munns] gave a talk at RustConf 2018, covering the basics of embedded systems, and how Rust's Zero Cost Abstractions are a perfect match for bare metal systems
+
+[paoloteti]: https://github.com/paoloteti
+
+[at RustConf 2018]: https://www.youtube.com/watch?v=t99L3JHhLc0
+[James Munns]: https://github.com/jamesmunns
+
+## Embedded Projects
+
+If you have an embedded project or blog post you would like to have featured in the Embedded WG Newsletter, make sure to mention it on the tracking issue for [the next newsletter], we would love to show it off!
+
+* Support for the Arduino MKRZERO board [has landed] in the [atsamd21-rs] repository! This is a Board Support Crate for various development board based on the Atmel/Microchip samd21 Cortex-M microcontroller
+* [Branan Riley] is working on the [fourth chapter] of his [Exploring Rust on Teensy] series, which will focus on using `futures-rs` to represent DMA transfers
+* [Emil Fresk] shared some shots of his next [Visual Inertial PCB], which includes an STM32 running control algorithms written in Rust!
+* [Jonathan Pallant] added a bare-metal [four-channel synthesizer] for his Monotron project, and even included a short [demo audio clip]!
+
+[fourth chapter]: https://twitter.com/branan/status/1038222375790161920
+[Exploring Rust on Teensy]: https://branan.github.io/teensy/
+[Branan Riley]: https://github.com/branan
+
+[Visual Inertial PCB]: https://twitter.com/korken89/status/1038494310830952449
+[Emil Fresk]: https://github.com/korken89
+
+[four-channel synthesizer]: https://twitter.com/therealjpster/status/1036298070022086656
+[Jonathan Pallant]: https://github.com/thejpster
+[demo audio clip]: https://thejpster.org.uk/beat.mp3
```

Comment:
Please could we swap thejpster.org.uk/beat.mp3 for https://www.youtube.com/watch?v=NvfyFhFK11g. Thanks.

Reply:
:+1: done!